src/streamsync/__init__.py

```diff
@@ -38,3 +39,24 @@ def init_state(state_dict: Dict[str, Any]):
     initial_state.user_state.state = {}
     initial_state.user_state.ingest(state_dict)
     return initial_state
+
+
+def init_ui() -> StreamsyncUIManager:
+    """
+    Initializes and returns an instance of StreamsyncUIManager. This manager provides 
+    methods to dynamically create and manage UI components in a Streamsync application.
+
+    The returned StreamsyncUIManager allows for the creation of application-wide, 
+    code-managed components during startup, ensuring that the set of components 
+    is initially accessible by all sessions.
+
+    Returns:
```

Comment:
You are using several documentation format (Google here and rEST in the other function). We should stick to one. I usually use the rEST one (https://stackoverflow.com/a/24385103) because I stick on this one usually because I am using [sphinx](https://www.sphinx-doc.org/en/master/) to write and generate API documentation.

Do you have one you prefer ?

Reply:
My apologies, I intended to use the rEST style, but forgot to rewrite it here. Will fix it.